src/borg/key.py

```diff
@@ -371,7 +371,7 @@ def encrypt_key_file(self, data, passphrase):
         iterations = PBKDF2_ITERATIONS
         key = passphrase.kdf(salt, iterations, 32)
         hash = hmac_sha256(key, data)
-        cdata = AES(is_encrypt=True, key=key).encrypt(data)
+        cdata = AES(key, b'\0'*16).encrypt(data)
```

Comment:
I guess it is not in scope of this PR to change anything related to that.
Also, this is done once for repokey/keyfile key generation, so what's the problem?
Via the random salt, the aes key is different for each repo/keyfile key.